source/user-guide/export/index.rst

```diff
@@ -43,4 +45,73 @@ It is possible to override the brief summary of metadata elements by creating a
    </metadata>
  </xsl:template>
 
-This template, when added to ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata-iso19139.xsl``, will replace the brief summary (produced by the brief template) with just one element, gmd:title.
+This template, when added to ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata.xsl``, will replace the brief summary (produced by the brief template) with just one element, gmd:title.
+
+In the case of iso19115-3.2018 there are the two files to be modified:
+
+1. geonetwork/WEB-INF/data/config/schema_plugins/iso19115-3.2018/layout/tpl-csv.xsl
+2. geonetwork/xslt/services/csv/csv-search.xsl
+
+For example, to export the following elements:
+
+``"uuid","title","cloud_coverage_percentage","category","date-creation"``
+
+In the first file, you have to add a similar line in between the abstract and category elements:
+::
+     <myfield>myfieldvalue</myfield>
+
+For example:
+
+::
+
+    <title>
+       <xsl:copy-of select="mdb:identificationInfo/*/mri:citation/*/cit:title"/>
+    </title>
+    <cloud_coverage_percentage>
+       <xsl:copy-of select="mdb:contentInfo/mrc:MD_ImageDescription/mrc:cloudCoverPercentage/gco:Real"/>
+    </cloud_coverage_percentage>
+    <category>
+      <xsl:value-of select="mdb:metadataScope/*/mdb:resourceScope/*/@codeListValue"/>
+    </category>
+
+About cloud coverage percentage, in particular, please remember to add the namespace if missing:   
+
+``xmlns:mrc="http://standards.iso.org/iso/19115/-3/mrc/2.0"``
+
+If not interested in other information, please comment all other lines except 
+
+``<xsl:copy-of select="gn:info"/>``
+
+About the second file, *csv-search.xsl*, to avoid printing automatically the 3 columns:
+
+``"schema","uuid","id",``
```

Comment:
An option could be to make this more generic. In tpl-csv.xsl, we could add those 3 columns by default. And they will be copied automatically by csv-search.xsl. So if you want to remove them, you could only change the schema template and no changes would be required here. What do you think, should we make a PR for that?

Reply:
In my opinion, it would be a great improvement because it could facilitate a lot the usage of that function also for a non-expert user.